internal/storage/storageutil/test_util.go

```diff
@@ -42,5 +42,13 @@ func GetDefaultStorageClientConfig() (clientConfig StorageClientConfig) {
 		ExperimentalEnableJsonRead: false,
 		AnonymousAccess:            true,
 		EnableHNS:                  false,
+		ReadStallRetryConfig: cfg.ReadStallGcsRetriesConfig{
+			Enable:              false,
```

Comment:
nit: Put these constant default values in a config somewhere as constants rather than here where they are harder to find.